### PR TITLE
Test against analyzer branch for SDK_AT_HEAD

### PR DESCRIPTION
**Pull request by MichaelRFairhurst**

### CODE REVIEW

File: tools/update_deps.sh [hunk 1]
```diff
@@ -6,6 +6,9 @@ echo
 
 echo Updating the sdk with depot_tools
 gclient sync
+cd sdk
+g checkout analyzer
```
Comment (stereotype441) [inline tools/update_deps.sh]: Do you mean `git checkout analyzer`?

Comment (MichaelRFairhurst) [reply tools/update_deps.sh]: ah yes! (I have `alias g=git` in my `.bashrc`...)

**Comment by MichaelRFairhurst** [comment]
ping @stereotype441

**Comment by stereotype441** [comment]
lgtm

**Pull request merged**